File: app/src/main/java/com/cvv/fanstaticapps/randomticker/activities/CancelActivity.java
package com.cvv.fanstaticapps.randomticker.activities;

import android.os.Bundle;

import com.cvv.fanstaticapps.randomticker.helper.TimerHelper;

import javax.inject.Inject;

public class CancelActivity extends BaseActivity {
    @Inject
    TimerHelper timerHelper;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        timerHelper.cancelNotificationAndGoBack(this, preferences);
    }
}

File: app/src/main/java/com/cvv/fanstaticapps/randomticker/activities/CancelActivity.kt
package com.cvv.fanstaticapps.randomticker.activities

import android.os.Bundle

import com.cvv.fanstaticapps.randomticker.helper.TimerHelper
import com.cvv.fanstaticapps.randomticker.prefs

import javax.inject.Inject

class CancelActivity : BaseActivity() {
    @Inject
    lateinit var timerHelper: TimerHelper

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        timerHelper.cancelNotificationAndGoBack(this, prefs)
    }
}
